crates/shim-protos/README.md

```diff
@@ -50,8 +50,17 @@ let resp = task_client.connect(context, &req)?;
 - [TTRPC shim server](./examples/ttrpc-server.rs)
 - [TTRPC client connect](./examples/connect.rs).
 
-The way to build the [TTRPC client connect](./examples/connect.rs) example:
+The way to build the example:
 ```bash
+## connect
+## build sync connect, client and server
 $ cargo build --example connect
 $ sudo ./connect unix:///containerd-shim/shim_socket_path.sock
+$ cargo build --example shim-proto-client
+$ cargo build --example shim-proto-server
+
+## build async connect, client and server
```

Comment:
### build async connect, client and server

Reply:
fixed